packages/odnnr/odnnr.2.0.3/files/install_keras.r

```diff
@@ -0,0 +1,8 @@
+# for keras and tensorflow, let's assume the user has installed them in
+# user-space as a prerequisite using 'pip3 install tensorflow keras'
+# here, we just test that the keras library can be loaded in R
+pkg <- 'keras'
+if (!require(pkg, character.only = TRUE)) {
+  install.packages(pkg, dependencies = TRUE, repos='http://cran.r-project.org')
```

Comment:
I know this pattern is everywhere in your R-related packages, and I’ve kinda let that slip under the rug for all those years but how is this supposed to work without network access? (sandbox)

Reply:
Isn't the sandbox too tight, if we cannot download listed things at install time?
Of course, R will also download all the required dependencies...